event_track_location_overlap/models/event_track_location.py

```diff
@@ -14,15 +12,14 @@ class EventTrackLocation(models.Model):
     overlappable = fields.Boolean(help="Can this location have simultaneous tracks?")
 
     @api.constrains("overlappable")
```

Comment:
You need also to split this for multi record support

Reply:
Ok, if I have understood it correctly, the last change would be correct?

Reply:
Yes.